integration/models/application.ts

```diff
@@ -63,9 +61,10 @@ export const application = (config: ApplicationConfig, appDirPath: string, appDi
         stderr: opts.detached ? fs.openSync(stderrFilePath, 'a') : undefined,
         log: opts.detached ? undefined : log,
       });
+      // const shouldRetry = () => proc.exitCode !== 0 && proc.exitCode !== null;
```

Comment:
Nit: Dead comment

Reply:
This should've been a todo - changing it in the new PR